index.js

```diff
@@ -3,16 +3,23 @@ new Vue({
   data: {
   	todos: [
   		{ text: 'test', done: false, id: Date.now() }
-  	]
+  	], 
+    activeTodos: [], 
+    completedTodos: []

```

Comment:
create two new variables here, `showActive` and set it to `false`, `showCompleted` and set it to `false`.